kubernetes/platform-services/Chart.yaml

```diff
@@ -78,5 +78,5 @@ dependencies:
     condition: alerting.enabled
   - name: kafka-topic-creator
     repository: "https://storage.googleapis.com/hypertrace-helm-charts"
-    version: 0.1.9
```

Comment:
The latest is 0.1.9 so revert back to it - https://github.com/hypertrace/kafka-topic-creator/releases

Reply:
Done